gluon-activations/gluon-activations.md

```diff
@@ -0,0 +1,239 @@
+
+## MXNet Activations Tutorial
+
+Deep neural networks encode a nonlinear, highly parametrized function from input to outputs. The nonlinearities that allow neural networks to be maximally expressive are referred to as activation functions. Over the course of the development of neural networks, several nonlinear activation functions have been introduced to make gradient-based deep learning tractable. 
```

Comment:
Sounds a bit to formal/'mathsy' for a beginner. "highly parametrized", "maximally expressive", "learning tractable". See if you can reword a bit. Can get more complex as you go on but start really simple.

Reply:
How about:

"Deep neural networks are a way to express a nonlinear function with lots of parameters from input data to outputs. The nonlinearities that allow neural networks to capture complex patterns in data are referred to as activation functions."

Reply:
👍